### PR TITLE
Fix #19909 and reactivate tests on ILC.

### LINKED ISSUE #19909: Reenable the tests ActiveIssue'd on 19909 this in Uap (not UapAot)
```
System.IO.MemoryMappedFiles.Tests -method System.IO.MemoryMappedFiles.Tests.DataShared

System.Net.Primitives.Functional.Tests -method System.Net.Primitives.Functional.Tests.EventSource_EventsRaisedAsExpected

System.Net.Security.Tests -method System.Net.Security.Tests.EventSource_EventsRaisedAsExpected
```

These tests are ActiveIssue'd against this issue but for Uap, not UapAot. Chances are they were intended to be ActiveIssued against some other process-related issue that prevents them from running in UWP (non-AOT) but since we've fix the aot-related issues, zombie'ing this issue to track reenabling these two tests.

### PR DESCRIPTION
**Pull request by AtsushiKan**
This may cause new failures on ILC runs because of new tests
running. I did not check the state of every test.

### CODE REVIEW

File: src/Common/src/Interop/Windows/kernel32/Interop.CreateProcess.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ internal partial class Kernel32
             IntPtr lpEnvironment,
             [MarshalAs(UnmanagedType.LPTStr)] string lpCurrentDirectory,
             STARTUPINFO lpStartupInfo,
```
Comment (danmosemsft) [inline src/Common/src/Interop/Windows/kernel32/Interop.CreateProcess.cs]: For some reason pinvoke.net has this one as `[In] ref`

**Comment by danmosemsft** [comment]
@mmitche OSX machines broken

File: src/Common/src/Interop/Windows/kernel32/Interop.CreateProcess.cs (hunk 1 above)
Comment (AtsushiKan) [inline src/Common/src/Interop/Windows/kernel32/Interop.CreateProcess.cs]: They're assuming STARTUPINFO defined as a struct. We define it as a class.

Comment (danmosemsft) [inline src/Common/src/Interop/Windows/kernel32/Interop.CreateProcess.cs]: Ah - yes thanks.

**Comment by AtsushiKan** [comment]
@dotnet-bot test Innerloop OSX10.12 Debug x64 Build and Test

**Comment by AtsushiKan** [comment]
@dotnet-bot Test Innerloop OSX10.12 Release x64 Build and Test

**Pull request merged**